fixtures/schema.js

```diff
@@ -51,31 +51,33 @@ var Physical = schema({
 
 var DMXValue = Number.min(0).step(1);  // max value depends on how many fine channels there are (255 if none, 65535 if one, etc.)
 
+var ChannelKey = String;  // channels in availableChannels
+var ChannelAliasKey = String;  // channel keys that are only defined inside other channels
+
 var Capability = schema({
   'range': Array.of(2, DMXValue),
   'name': NonEmptyString,
   '?menuClick': ['start', 'center', 'end', 'hidden'],
   '?color': Color,
   '?color2': Color,
   '?image': NonEmptyString,
+  '?switchToChannels': Array.of(1, Infinity, ChannelKey), // same array length as Channel.switchesChannels
```

Comment:
So you wouldn't declare switching channels in the channel itself but only in each capability? But then we had to write the same switching channel alias n times in n capabilities which is redundant and therefore leads to mistakes and worse maintainability.

Though, I agree that it would be more intuitive and more clear.